store/server/features/leads/mutation.ts

```
@@ -0,0 +1,130 @@
+    onError: (error: any) => {
+      // Provide user-friendly error messages
+      // eslint-disable-next-line
+      let errorMessage = 'Failed to create lead.';
+
+      if (error?.message) {
+        if (error.message.includes('Tenant ID not found')) {
+          errorMessage = 'Authentication error. Please log in again.';
+        } else if (error.message.includes('Unauthorized')) {
+          errorMessage = 'You are not authorized to create leads.';
+        } else {
+          errorMessage = error.message;
```

ESLint

[error] 'errorMessage' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)